include/openthread/thread.h

```diff
@@ -305,7 +305,7 @@ otError otThreadSetJoinerAdvertisement(otInstance *   aInstance,
 #define OT_JOINER_ADVDATA_MAX_LENGTH 64 ///< Maximum AdvData Length of Joiner Advertisement
 
 /**
- * Get the Thread Child Timeout used when operating in the Child role.
+ * Gets the Thread Child Timeout seconds used when operating in the Child role.
```

Comment:
nit: we typically mention units as follows.
```suggestion
 * Gets the Thread Child Timeout (in seconds) used when operating in the Child role.
```